### DISCUSSION TITLE
DataCite 4.4 new fields for publications and conferences

### DISCUSSION THREAD

**Discussion by fenekku** [inveniosoftware/product-rdm#150] [Ideas]
(transcribing from feedback of librarians at NU)

As a researcher, my citations don't look quite right because the conference location part is missing.

Expected:
1. Carson M, Gonzales S, Shaw P, Schneider D, Holmes K. A Library-based Data Retrieval and Management Training Program for Clinical Researchers.  **Research Data Alliance 17th Plenary Meeting; April 20-23; Edinburgh, Scotland (Virtual)**. Prism. Northwestern University; 2021.

Got:

![image](https://user-images.githubusercontent.com/1932651/129363423-f324b862-97b1-4659-87ef-5b187b2b46a2.png)

## Some floated solutions

- Resource type dependent field. For example, conference poster should prompt for conference information. Dev: we are not equipped to deal with that yet
- Custom field. Dev: would need to be able to connect it to citation system

priority: low

[a comment by an automated account is not shown]

**Comment by lnielsen** [comment]
Zenodo has fields for publications and conferences. We'll need either custom fields or extending the core data model to support the new DataCite 4.4 fields

**Comment by fenekku** [comment]
If those are part of Datacite 4.4, then we could add them to the core metadata model and that will be it. They should get picked up by citation generators and we won't need any other machinery.

**Comment by lnielsen** [comment]
Yes, we should take a look. Also, we'll need to update to DataCite v4.5
